boards/others/neorv32/CMakeLists.txt

```diff
@@ -18,7 +18,7 @@ if((CONFIG_BOARD_NEORV32) AND (CONFIG_BUILD_OUTPUT_BIN))
 
     set_property(GLOBAL APPEND PROPERTY extra_post_build_commands
       COMMAND ${IMAGE_GEN}
```

Comment:
I take it this builds a rom image you can use with vhdl? Neat!

Reply:
Correct. See https://stnolting.github.io/neorv32/ug/#_installing_an_executable_directly_into_memory for details.